### PR TITLE
Flit integration 

### PR DESCRIPTION
**Pull request by ibaldin**
Joining the 21st century so this is built using Flit.

### CODE REVIEW

**Review by kthare10 — APPROVED** [review]
Looks good!

**Review by sajith — APPROVED** [review]
The packaging parts look good to me!  Except for a comment (inline) about specifying optional dependencies, and a couple of notes:

- `requirements.txt` can be now removed, unless there is a reason to keep it.
- `MANIFEST.in` is not necessary anymore, and should be safe to remove.  The new tools aim to obsolete manifest files. Flit explicitly states that "data files within a package directory are automatically included" in https://flit.pypa.io/en/latest/rationale.html.

I am not familiar with the shell scripts under "util", so I'm ignoring those.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"matplotlib >= 3.6.0",
+	"neo4j >= 5.2.0",
+	"networkx >= 2.8.7",
+	"networkx_query >= 1.0.1",
+	"PyYAML >= 6.0",
+	"recordclass >= 0.17.5",
+	"requests >= 2.28.1",
+	"lxml >= 4.9.2"
+	]
+
```
[the remark this answers is not in the document]
Comment (ibaldin) [reply pyproject.toml:26]: I added optional dependencies and removed the MANIFEST.in.

Without a requirements.txt how does one build a virtenv to work in?

[the remark this answers is not in the document]
Comment (sajith) [reply pyproject.toml:26]: This appears to work, without a requirements.txt:

```console
$ cd InformationModel
$ mkvirtualenv -i .[test] infomodel
$ workon infomodel
```

I rarely use virtualenvwrapper though, so I am not certain that this is the right answer.  I simply use `python -m venv venv --upgrade-deps; source ./venv/bin/activate`.

**Pull request merged**